host-controller/src/main/java/org/jboss/as/host/controller/Main.java

```diff
@@ -74,8 +74,6 @@ public static void main(String[] args) throws IOException {
 
         // Grab copies of our streams.
         final InputStream in = System.in;
-        //final PrintStream out = System.out;
-        //final PrintStream err = System.err;
```

Comment:
Should we add a comment here as to why we're no longer capturing these or potentially just delete them?

Reply:
:) We crossed the messages, those were only the removal of two lines already commented that don't provide anything about what they are commented

Reply:
Looking at https://github.com/wildfly/wildfly-core/commit/008be9c6#diff-c2b144af158144bf34cc44a678172b1370f626d5a23e0d0e50f4cc124b516fed, I'm pretty sure those vars had been ultimately unused. AFAICT that commit didn't remove any ultimate use; it just stopped passing the values through method calls. 

I probably wanted to leave the var declarations commented as a kind of tombstone because the code would no longer "Grab copIES of our streamS." Bad choice on my part; it ultimately was just confusing.

That said, @yersan perhaps change that "// Grab copies of our streams" comment to "// Get a ref to the original stdin before we install JBoss Stdio"

Reply:
lol I definitely read the diff wrong there :)